conf.d/pure.fish

```diff
@@ -15,6 +15,7 @@ _pure_set_default pure_color_prompt_on_success $pure_color_success
 _pure_set_default pure_symbol_git_unpulled_commits "⇣"
 _pure_set_default pure_symbol_git_unpushed_commits "⇡"
 _pure_set_default pure_symbol_git_dirty "*"
+_pure_set_default pure_color_git_pending_commits $pure_color_info
```

Comment:
I still think we should use two variables🖖 I thought @rafaelrinaldi agreed with me on this: https://github.com/rafaelrinaldi/pure/issues/130#issuecomment-453797592